streaming/vision/convert/coco.py

```diff
@@ -235,7 +235,11 @@ def main(args: Namespace) -> None:
         else:
             dataset = each(dataset, shuffle)
 
-        with MDSWriter(split_out_dir, fields, args.compression, hashes, args.size_limit) as out:
+        with MDSWriter(local=split_out_dir,
```

Comment:
nit: `split_dir` in sync with others ?

Reply:
other variables are named `split_..._in_dir`